pmkoalas/models/pnformatter.py

```diff
@@ -0,0 +1,79 @@
+
+import math
+from pmkoalas.models.petrinet import PetriNetDOTFormatter
+from pmkoalas.models.dotutil import exportDOTToImage
+
+
```

Comment:
So you have a separate file for half of the formatting code? Do you want to move all formatting code to this function or do you want to move this code to petrinet?

Reply:
Definitely should be separate to petrinet